content/2_Bronze/Intro_Greedy.mdx

```diff
@@ -179,3 +180,63 @@ programmers refer to this as "Proof by AC," or "Proof by Accepted."
 <Problems problems="general" />
 
 <!-- Anything else major that needs to be added? Bronze Ad Hoc is not very complicated. -->
+	
+<Quiz>
+  <Quiz.Question>
+    What's a greedy algorithm?
+    <Quiz.Answer>
+      An algorithm which backtracks in order to find the most optimal solution.
+      <Quiz.Explanation>
+        Incorrect. A greedy algorithm doesn't backtrack. Instead, it directly constructs an answer.
```

Comment:
i mean there are greedy backtracking algorithms (like the one in AC - independent set I think?)

Reply:
so something along the lines of it might not necessarily backtrack would be more correct

Reply:
sure but in general for 99.99% of the time he got greedy right

Reply:
i think its fine youre pointing out something that is a very specific case

Reply:
yeah idk, i was just taking out sections from the module:
![image](https://user-images.githubusercontent.com/81596991/132635404-768a5819-ef4b-42ea-b3ec-7fb56624e1bf.png)

Reply:
correct me if i'm wrong, but taking back a choice is different from backtracking (which is a method of search)?

Reply:
> correct me if i'm wrong, but taking back a choice is different from backtracking (which is a method of search)?

i thought backtracking was kind of it's own algorithm? you might be right, i think i worded the answer unclearly

Reply:
i changed it to "take back choices", but if there's anything else, just lmk :D